### PR TITLE
[114221689] add highlight

### PR DESCRIPTION
**Pull request by andela-sdamian**
add highlights when creating events
include pictures in each highlight item
display highlight on the preview page
display highlight on the event details page

### CODE REVIEW

File: app/assets/stylesheets/events.scss [hunk 1]
```diff
@@ -8,8 +8,8 @@ $white-color: #000;
   margin-top: 50px;
   padding-bottom: 20px;
   input, select{
-    border: 1px solid #CCCCCC;
-    background-color: #ffffff;
+    //border: 1px solid #CCCCCC;
+    //background-color: #ffffff;
```
Comment (andela-iarisah) [inline app/assets/stylesheets/events.scss]: if this styling is no longer in use, it'd be best to delete it

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -89,12 +102,16 @@ def event_params
...
+                                  [:id, :_destroy, :day, :title, :description,
+                                   :start_time, :end_time, :image,
+                                   :image_title],
                                   event_staffs_attributes:
                                     [:user_id, :role])
   end
 
   def set_events
-    @event = Event.find_by_id(params[:id])
+    @event = Event.find_by(id: params[:id])
```
Comment (andela-iarisah) [inline app/controllers/events_controller.rb]: it'll be more optimal to have `Event.find(params[:id])`

File: app/views/events/_form.html.erb [hunk 3]
```diff
@@ -171,9 +194,9 @@
 <!-- <iframe id="frame" scrolling="yes" src="30" width="80%" height="500px" frameborder="0" style="margin:0 auto;">
   </iframe> -->
 
-  <div class="" id="preview-event-div" style="width: 80%; height: 500px; margin: 0 auto 1000px auto">
+  <div class="" id="preview-event-div" style="width: 80%; height: 500px; margin: 0 auto 5% auto; overflow: auto;">
     <% @current_page = "new" %>
-    <%= render partial: "events/eventdetails" %>
+    <%= render "eventdetails" %>
```
Comment (andela-iarisah) [inline app/views/events/_form.html.erb]: 👍

File: test/models/highlight_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class HighlightTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (andela-iarisah) [inline test/models/highlight_test.rb]: if there is a `spec` directory, there is no need to also have a `test` directory

**Comment by andela-iarisah** [comment]
good work overall

File: app/controllers/events_controller.rb (hunk 2 above)
Comment (andela-sdamian) [inline app/controllers/events_controller.rb]: make sense :+1:

File: test/models/highlight_test.rb (hunk 4 above)
Comment (andela-sdamian) [inline test/models/highlight_test.rb]: make sense :+1:

**Comment by andela-iokonkwo** [comment]
Due to the different houndCI issues and PR description not following convention, I will be closing this PR.

**Pull request closed without merging**